management-interface.go

```diff
@@ -289,6 +292,279 @@ func NetworkHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "network.html", state)
 }
 
+//getNetworkSSID -gets the ssid of the network associated with this id
+func getNetworkSSID(networkID string) (string, error) {
+	var out []byte
+	err := error(nil)
```

Comment:
these two lines aren't necessary, just change the `exec.Command` line to `out, err := exec.Command(...)`